charts/nextcloud/templates/nginx-config.yaml

```diff
@@ -44,7 +32,7 @@ data:
         }
 
         server {
-            listen 80;
+            listen {{ .Values.nginx.containerPort | default "80" }};
```

Comment:
Sorry to miss this before it merged, but I'm curious why not just `.Values.nextcloud.containerPort`? I'm not seeing a reason why a new variable was necessary, and now it seems like it's necessary to set both variables. The `deployment.yaml` template still uses `.Values.nextcloud.containerPort` for the `containerPort` of the nginx container.

Reply:
This was the case in my original PR, but we discussed this here https://github.com/nextcloud/helm/pull/440#discussion_r1314479033 and ended up with an extra var.

Reply:
Thanks for pointing out the previous discussion. I don't follow the argument for granularity here, but if that feels right to you both then we should keep it.

The template update to fall back to the primary container port variable seems convenient, but I'm fine to leave things as they are.

It's only because I didn't understand any current or future scenarios where these variables would *ever* have different values, and didn't see the previous discussion, that I suggested just removing the nginx one.